frontend/fetch-version.ts

```diff
@@ -31,6 +45,9 @@ async function main() {
   console.error(gitTagResponse.stderr);
   const response = await github;
   const data = await response.json();
+  if (!response.ok) {
+    console.error(data);
```

Comment:
Should this be logged in console?

Reply:
The script is invoked in the cli directly and lives as a standalone script, therefore it is the easiest way to print the error messages as `stderr`.